modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,286 @@
+step 1b.  Only the alphanumeric information that proceeds the '.net/' within the URL is needed.
+====
+--
+. Add the tags necessary for subnet discovery:
+
+.. Add the following {Key: Value} tag to the VPC hosting the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster, replacing '{Cluster Infra ID}' with the Infra ID specified in step 1a:
+
+
+* kubernetes.io/cluster/${Cluster Infra ID}:owned
+
+.. Add the following ELBv2 {Key: Value} tags to the private subnets and (optional) public subnets:
+
+* Private subnets: kubernetes.io/role/internal-elb:1
+* Public subnets: kubernetes.io/role/elb:1
++
+[NOTE]
+====
+Internet-facing and internal load balancers will be created within the AZ to which these subnets belong.
+====
++
+For more information on adding tags to AWS resources (including VPCs and subnets), see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
++
+[IMPORTANT]
+====
+ELBv2 resources (such as ALBs and NLBs) created by ALBO do not inherit custom tags set for
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+clusters. Tags must be set separately for these resources.
+====
++
+. Log into the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster as the cluster-admin and create a new project using the following command:
++
+[source, terminal]
+----
+$ oc new-project aws-load-balancer-operator
+----
++
+. Link the AWS IAM role created in step 2a with the ALBO service account and create a Kubernetes Secret for the ALBO:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+stringData:
+  credentials: |
+    [default]
+    role_arn = <aws-load-balancer-operator-role-arn><1>
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+----
+<1> Replace <aws-load-balancer-operator-role-arn>' with the AWS IAM role created in step 2a.
++
+. Create ALBO:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  upgradeStrategy: Default
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.0
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: aws-load-balancer-operator.v1.0.0
+----
++
+. Create an AWS ALBC:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1
+kind: AWSLoadBalancerController
+metadata:
+  name: cluster
+spec:
```

Comment:
If the manual tagging was done before, `AWSLoadBalancerController` needs to set the [subnetTagging field](https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/tutorial.md#subnettagging) to `Manual`. Otherwise `Auto` mode is set by default.

Reply:
Yes, `subnetTagging` needs to be set to `Manual` if it was done manually before (like in your document). Otherwise the operator will try to tag the subnets and in the case of the BYO VPC it can mess things up.

> Also, do you think it is necessary to tell the reader that Auto is set by default?

Normally the user can use `oc explain AWSLoadbalancerController` command and get the description of each field, with mentions of what is set by default. So I would say no, it's not strictly necessary to mention the default here.

Reply:
Got it, thanks @alebedev87.